src/components/TheHeader.vue

```diff
@@ -96,9 +101,15 @@ const signupButtonLink = computed(() => {
 
 <style scoped>
 @media screen and (min-width: 960px) {
-  .desktop-logo {
+  .logo-container {
+    display: block;
     position: absolute;
-    left: calc(50% - 60px);
+    left: 50%;
+    transform: translateX(-50%);
```

Comment:
Is this to move the logo back to the right? It looks like perhaps if we kept the `calc()` call to lower the `left` amount like we previously had it that it might remove the need for this?